packages/web-pkg/src/services/eventBus.ts

```diff
@@ -7,10 +7,10 @@ export class EventBus {
     this.topics = new Map()
   }
 
-  public subscribe(topic: string, callback: (data?: unknown) => void): string {
+  public subscribe<T = unknown>(topic: string, callback: (data?: T) => void): string {
     const subscription = {
       token: uuid.v4(),
-      callback
+      callback: callback as (data?: unknown) => void
```

Comment:
why can't the argument be typed better?

Reply:
As discussed, currently the most convenient approach this way.